src/map/battle.cpp

```diff
@@ -1525,7 +1525,11 @@ int64 battle_calc_damage(struct block_list *src,struct block_list *bl,struct Dam
 
 		if (flag&BF_MAGIC && bl->type == BL_PC && sc->data[SC_GVG_GIANT] && sc->data[SC_GVG_GIANT]->val4)
 			damage += damage * sc->data[SC_GVG_GIANT]->val4 / 100;
-
+		
```

Comment:
This is a Renewal only change, so please add `#ifdef RENEWAL`

Reply:
I dont think that's necessary cause the race2 mobs were added in the re/mob_race2_db.txt

Reply:
Oh, that's right, nevermind